classic-bg-ui/setup-classic-bg-ui.tp2

```diff
@@ -0,0 +1,10 @@
+BACKUP ~weidu_external/backup/classic-bg-ui~
+
+AUTHOR ~Author Name (author's email address)~
```

Comment:
Probably this should be filled with non-placeholders before merging/released.

Reply:
It has to have a check whether SCS is installed and flip the flag at the beginning of ui.menu accordingly.

I am not super into weidu that does exactly what you do yourself now (copies files from one place to another), because you then have to suffer through its integrity checks, but it's probably ok to add, for completion's sake if anything.